api/src/modules/sourcing-locations/dto/materials-query.sourcing-location.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsBoolean, IsEnum, IsOptional, IsString } from 'class-validator';
+
+export enum MATERIALS_ORDER_BY_VALUES {
+  COUNTRY = 'country',
+  BUSINESS_UNIT = 'businessUnit',
+  PRODUCER = 'producer',
+  T1SUPPLIER = 't1Supplier',
+  MATERIAL = 'material',
+  LOCATION_TYPE = 'locationType',
+}
+
+export class GetSourcingMaterialsQueryDto {
+  @ApiProperty({ required: false, enum: MATERIALS_ORDER_BY_VALUES })
+  @IsOptional()
+  @IsEnum(MATERIALS_ORDER_BY_VALUES, {
+    message:
+      'Available columns for orderBy: country, material, producer, t1Supplier, locationType',
+  })
```

Comment:
```suggestion
  @IsEnum(MATERIALS_ORDER_BY_VALUES, {
    message: `Available columns for orderBy: ${Object.values(
      MATERIALS_ORDER_BY_VALUES,
    ).join(', ')}`,
  })
```

Reply:
Also: your enum has 6 values, but the error message only has 5 - which is the perfect example of why my change suggestion makes sense. tests fail once you change it, so mind that too